src/index.css

```
@@ -1204,3 +1235,96 @@ body {
+  z-index: 40;
+  overflow: hidden;
+  pointer-events: none;
+}
+
+.fx-comet {
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  margin: -5px 0 0 -5px;
+  border-radius: 999px;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)